<?xml version="1.0" ?>
<sdf version="1.5">
  <world name="default">
    <!-- A global light source -->
    <include>
      <uri>model://sun</uri>
    </include>
    <!-- A ground plane -->
    <include>
      <uri>model://ground_plane</uri>
    </include>

    <!-- A testing model that includes the AWL16 sensor model -->
    <model name="awl16">
      <include>
        <uri>model://awl16</uri>
      </include>
      <static>true</static>
      <pos>0 0 0</pos>

    </model>

  </world>
</sdf>